tests/iface_loopback_action/test_iface_loopback_action.py

```diff
@@ -97,13 +98,15 @@ def test_loopback_action_reload(request, duthost, localhost, ptfadapter, ports_c
             reload_types = ["reload", "cold", "fast", "warm"]
             reboot_type = random.choice(reload_types)
         if reboot_type == "reload":
-            config_reload(duthost, safe_reload=True, check_intf_up_ports=True)
+            config_reload(duthost, safe_reload=True)
         else:
             reboot(duthost, localhost, reboot_type)
             pytest_assert(wait_until(300, 20, 0, duthost.critical_services_fully_started),
                           "All critical services should be fully started!")
-            pytest_assert(wait_until(300, 20, 0, check_interface_status_of_up_ports, duthost),
-                          "Not all ports that are admin up on are operationally up")
+        # Wait 180s for the rif counter to initialize
+        time.sleep(180)
```

Comment:
This doesn't make sense to me, sleep of 3 minutes, just to get the counters updates, can we modify the polling rate?

Reply:
@roy-sror Confirmed with @Junchao-Mellanox again. Cold reboot requires the hard coded 180s, by for the other types, it may not require so long time. I'll update the fix with a wait_until to check when the rif counters are available.

Reply:
Fixed.